lib/modules/versioning/api.ts

```diff
@@ -17,6 +17,7 @@ import * as npm from './npm';
 import * as nuget from './nuget';
 import * as pep440 from './pep440';
 import * as poetry from './poetry';
+import * as redhat from './redhat';
```

Comment:
```suggestion
import redhat from './redhat';
```

We should start using default export 🤔